include/sacn/receiver.h

```diff
@@ -93,13 +92,32 @@ typedef struct SacnRemoteSource
   char name[SACN_SOURCE_NAME_MAX_LEN];
 } SacnRemoteSource;
 
+/*! Information about a sACN source that was found. */
+typedef struct SacnFoundSource
+{
+  /*! The Source Description. */
+  SacnRemoteSource info;
+  /*! The address from which we received these initial packets. */
+  EtcPalSockAddr from_addr;
+  /*! The per-universe priority. */
+  uint8_t priority;
+  /*! The DMX (startcode 0) data. */
+  uint8_t values[DMX_ADDRESS_COUNT];
+  /*! The count of valid values. */
+  size_t values_len;
+  /*! Whether or not we only saw startcode 0 packets with the preview flag set. */
+  bool values_were_preview;
```

Comment:
How about just "preview", more concise and still conveys the same amount of information.